spec/models/user_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe User do
+  let!(:user) { User.create(name: "Markus",
+                           email: "molsen13@gmail.com",
+                           password: "password") }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 6, 50 should be on its own line.